xarray/core/common.py

```diff
@@ -459,10 +459,13 @@ def where(self, cond):
         Parameters
         ----------
         cond : boolean DataArray or Dataset
+        drop : boolean specifying whether coordinate elements composed of only
```

Comment:
I think it's more accurate to say "coordinate labels that only correspond to NA values should be dropped" -- the coordinates are not NaN, but rather the data.

also, this is not quite formatted correctly (the type should be on its own):
```
drop : boolean, optional
    Some text description
```